apps.json

```
@@ -357,5 +357,15 @@
     "founders_username": ["colin-", "ba"],
     "logo": "",
     "screenshots": ["https://i.imgur.com/hG2sEYt.png"]
+ },
+   {
+    "name": "Buoy",
+    "slug": "buoy",
+    "description": "Get real-time notifications about keywords on Farcaster. Powerful filters help you cut through the noise. Stay on top of the conversation with Buoy.",
+    "categories": ["search, notifications, discovery, tools"],
```

@pfista do you mind changing this line to an array?
```suggestion
    "categories": ["search", "notifications", "discovery", "tools"],
```

---

No problem, will do!